src/mon/NVMeofGwMap.cc

```diff
@@ -819,7 +825,7 @@ int NVMeofGwMap::blocklist_gw(
     bool rc = addr_vect.parse(&str[0]);
```

Comment:
rc implies return_code.  return_codes usually use 0 to denote success.  entity_addrvec_t does not.  How about we just name this boolean 'success' rather than 'rc'?

Reply:
yes, makes sense